.github/workflows/linters.yml

```diff
@@ -10,8 +10,11 @@ jobs:
       - uses: psf/black@stable
         with:
           options: "--check --diff --verbose"
+          version: 24.1.1
   ruff:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - uses: chartboost/ruff-action@v1
+        with:
+            version: 0.1.4
```

Comment:
This is now `0.1.15`.

https://github.com/cfpb/regtech-data-validator/blob/45bb608b885f991e60a2439c64cdd12e7fb7d938/pyproject.toml#L21

Reply:
I've changed things up taking your annoyance (which I agree with).  I changed the linters to instead do what the pytest does: installs poetry and runs `poetry run black\ruff` instead of relying on actions of said libraries.  I put the linters into their own group in poetry so that the linter action can do `poetry install --only linters` (to keep from unnecessarily installing packages it doesn't need).  Give a look over of those changes.   This keeps everything running off of one poetry pyproject.toml instead of having versions in two different places.  

If we're good with this, I can write stories for the other repos to switch to the same linting actions.